bin/migrate.sh

```diff
@@ -167,6 +167,7 @@ yarn install
 if yarn workspace @datadog/datadog-ci-base build && yarn workspace "$PLUGIN_PKG" build && yarn workspace "$PLUGIN_PKG" lint --fix; then
   echo "Linting passed, running knip..."
   yarn knip || true
+  yarn
```

Comment:
Tiny fix: `yarn.lock` is not updated after running knip, so we need to run `yarn` now